Braintree.xcodeproj/project.pbxproj

```diff
@@ -4547,7 +4589,7 @@
 				SUPPORTS_MACCATALYST = NO;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "DEBUG $(inherited)";
 				SWIFT_EMIT_LOC_STRINGS = YES;
-				SWIFT_INSTALL_OBJC_HEADER = NO;
+				SWIFT_INSTALL_OBJC_HEADER = YES;
```

Comment:
This setting was causing the build to fail because the VisaCheckout module is in Obj-C so we needed to set this to yes. Stephanie and I paired on this and got it sorted. :)